python/src/spark_rapids_ml/classification.py

```diff
@@ -860,6 +860,8 @@ class LogisticRegression(
         If False, always uses dense array. This is favorable if the majority of VectorUDT vectors are DenseVector.
         If True, always uses sparse array. This is favorable if the majority of the VectorUDT vectors are SparseVector.
         Note this is only supported in spark >= 3.4.
+    float32_inputs: bool (default=True):
```

Comment:
We have opted to not add this as a documented constructor argument.   And document only here:  https://nvidia.github.io/spark-rapids-ml/compatibility.html#single-vs-double-precision-inputs  
Should we revisit that?

Reply:
Did not notice that. Revised to be consistent with compatibility.html.